CONTRIBUTING.md

```diff
@@ -102,13 +106,13 @@ To update the top nav bar web component:
 1. In https://github.com/Qiskit/web-components, run `npm install` then `npm run build`.
 2. There should be a file created at the root of the web components repository called `experimental-bundled-ui-shell.js`. Copy its contents into these files in this theme repository:
    1. `src/qiskit_sphinx_theme/pytorch/static/js/web-components/top-nav-bar.js`
-   2. `src/qiskit_sphinx_theme/furo/static/js/web-components/top-nav-bar.js`
+   2. `src/qiskit_sphinx_theme/theme/qiskit-sphinx-theme/static/js/web-components/top-nav-bar.js`
```

Comment:
100% agreed. But yes, sphinx-theme-builder is extremely opinionated

Reply:
hmm that's annoying 🤔 would it be an option to remove a folder layer higher up? So have something like this:

```
src
 ├── assets 
 ├── ecosystem
 ├── pytorch
 └── theme 
      └── qiskit-sphinx-theme
```

or would that mess things up as well?

Reply:
That would break how Python packages work. We could no longer import `qiskit_sphinx_theme`. And I think sphinx-theme-builder would complain also too hah